armi/reactor/converters/axialExpansionChanger.py

```diff
@@ -85,7 +85,12 @@ def performPrescribedAxialExpansion(
         self.axiallyExpandAssembly(thermal=False)
 
     def performThermalAxialExpansion(
-        self, a, tempGrid: list, tempField: list, setFuel=True
+        self,
+        a,
+        tempGrid: list,
+        tempField: list,
+        setFuel: bool = True,
+        updateNDensForRadialExp: bool = True,
```

Comment:
Please include `updateNDensForRadialExp` documentation in the `Parameters` section below of the docstring and note when False might be desirable in that doc. The comment below on this being set to False mainly for testing is a good one to note as well.

Reply:
Addressed. https://github.com/terrapower/armi/pull/845/commits/e3c3e2d148bb34dce22a55c031420ec2c2939011